Source/WebCore/page/Navigator.cpp

```diff
@@ -247,11 +263,29 @@ void Navigator::initializePluginAndMimeTypeArrays()
 
     auto [publiclyVisiblePlugins, additionalWebVisiblePlugins] = frame->page()->pluginData().publiclyVisiblePluginsAndAdditionalWebVisiblePlugins();
 
+    bool hasWebKitBuiltinPDFPlugin = frame->page()->pluginData().builtInPDFPlugin().has_value();
+    bool reportDummyPDFPlugin = frame->loader().client().canShowMIMEType("application/pdf"_s);
+
+    m_pdfViewerEnabled = hasWebKitBuiltinPDFPlugin || reportDummyPDFPlugin;
```

Comment:
Is `hasWebKitBuiltinPDFPlugin` really needed? Wouldn't `frame->loader().client().canShowMIMEType("application/pdf"_s)` be true also if with had the builtin PDF plugin?

Reply:
Yes. We are trying to distinguish between the case where we have a real plugin (macOS), and all the other platforms that either use a separate PDF view (iOS, WebKitLegacy), or PDFJS. So this flag is needed so we can populate